recipes/slamdunk/meta.yaml

```diff
@@ -42,6 +42,7 @@ requirements:
     - curl >7.59
     - xz
     - libdeflate
+    - openssl=1.0
```

Comment:
This should probably be removed and just fixed by rebuilding as it is pinned centrally.

Reply:
This needs to be removed. Openssl is pinned via the conda-forge-pinning repository.